### PR TITLE
Inactive socket fix

### PR DESCRIPTION
**Pull request by szokodiakos**

### CODE REVIEW

File: src/pages/display.tsx [hunk 1]
```diff
@@ -127,7 +136,10 @@ class Display extends Component<DisplayProps, DisplayComponentState> {
       case views.DISPLAY_WAITING_TO_START:
         return <WaitingToStart />
       case views.DISPLAY_GAME:
-        return <Game socket={socket!} onGameOver={this.gameOver} />
+        if (!socket) {
+          throw new Error("Socket is not available")
```
Comment (sarukuku) [inline src/pages/display.tsx]: What does this render if we don't have the socket here?

Comment (szokodiakos) [reply src/pages/display.tsx]: it shouldn't occur ever but I don't like to use the `!` operator in TS as it makes the code unreliable. We can add an error boundary and render a basic "uh oh something happened" page if any error is thrown.

Comment (sarukuku) [reply src/pages/display.tsx]: Sounds better. It looks a bit hazard to throw in a switch without a default case or catch. Looks like we don't know what to do if shit hits the fan.

Comment (szokodiakos) [reply src/pages/display.tsx]: Added an error page and if the socket is falsy we'll render the "Something went wrong"

Comment (sarukuku) [reply src/pages/display.tsx]: 👍

**Pull request merged**